docs/contracts/introduction.md

```diff
@@ -19,41 +19,41 @@ This modular design also helps in making the codebase easier to navigate and und
-| [`@lukso/lsp25-contracts`](https://github.com/lukso-network/lsp25-contracts)                                   | <a href="https://www.npmjs.com/package/@lukso/lsp25-contracts" target="_blank" rel="noopener noreferrer"><img style={{verticalAlign: 'middle'}} alt="npm badge" class="shield-badge" src="https://img.shields.io/npm/v/@lukso/lsp25-contracts.svg?style=flat&label=NPM&logo=npm"/></a>                                   |
-| [`@lukso/lsp26-contracts`](https://github.com/lukso-network/lsp26-contracts)                                   | <a href="https://www.npmjs.com/package/@lukso/lsp26-contracts" target="_blank" rel="noopener noreferrer"><img style={{verticalAlign: 'middle'}} alt="npm badge" class="shield-badge" src="https://img.shields.io/npm/v/@lukso/lsp26-contracts.svg?style=flat&label=NPM&logo=npm"/></a>                                   |
-| [`@lukso/universalprofile-contracts`](https://github.com/lukso-network/universalprofile-contracts)             | <a href="https://www.npmjs.com/package/@lukso/universalprofile-contracts" target="_blank" rel="noopener noreferrer"><img style={{verticalAlign: 'middle'}} alt="npm badge" class="shield-badge" src="https://img.shields.io/npm/v/@lukso/universalprofile-contracts.svg?style=flat&label=NPM&logo=npm"/></a>             |
+| Package                                                                                                                              | Version                                                                                                                                                                                                                                                                                                                  |
+| :----------------------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`@lukso/lsp10-contracts`](https://github.com/lukso-network/tree/develop/packages/lsp10-contracts)                                   | <a href="https://www.npmjs.com/package/@lukso/lsp10-contracts" target="_blank" rel="noopener noreferrer"><img style={{verticalAlign: 'middle'}} alt="npm badge" class="shield-badge" src="https://img.shields.io/npm/v/@lukso/lsp10-contracts.svg?style=flat&label=NPM&logo=npm"/></a>                                   |
```

Comment:
Half of the links was fixed. This part still has broken links that lead to 404.
Search by `https://github.com/lukso-network/tree/` and you'll see all links that should be modified.